scripts/release.sh

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env bash
+set -ex
+
+cd "$(dirname "${0}")"/..
+source ./scripts/version.sh
+
+SRC_DIR=${SRC_DIR:-$PWD}
+DOCKER=${DOCKER:-docker}
+GO=${GO:-go}
+GOPATH=${GOPATH:-$(go env GOPATH)}
+
+# TAG=v1.0.0
+
+RELEASE_DIR=${GOPATH}/src/github.com/flannel-io/cni-plugin/release-"${TAG}"
+OUTPUT_DIR=${GOPATH}/src/github.com/flannel-io/cni-plugin/dist
+
+# Always clean first
+rm -rf "${SRC_DIR:/}""${OUTPUT_DIR}"
+rm -rf "${SRC_DIR:/}""${RELEASE_DIR}"
+mkdir -p "${SRC_DIR}"/"${RELEASE_DIR}"
+mkdir -p "${SRC_DIR}"/"${OUTPUT_DIR}"
```

Comment:
Why `${SRC_DIR}`? Shouldn't it be enough with `${RELEASE_DIR}` and `${OUTPUT_DIR}`?

Reply:
@rosskirkpat is gonna clean this up a bit and we'll be done

Reply:
I removed the unnecessary use of `SRC_DIR` in scripts/release.sh and Dockerfile